tests/conftest.py

```diff
@@ -1,3 +1,8 @@
+# Adding pytest base dir to Python system path
+import site
+from os.path import dirname, abspath
+site.addsitedir(dirname(abspath(__file__)))
```

Comment:
this is still required for various common... import in various test cases and in plugins within this file

Reply:
Can you mention that in a comment in the file so we don't forget?

Reply:
done